examples/06_ken_embeddings.py

```diff
@@ -1,86 +1,101 @@
+they improve our model's accuracy (spoiler: they do).
 
-We will see that these embeddings of common entities significantly
-improve our results.
 
 .. note::
     This example requires `pyarrow` to be installed.
 
 .. [#] https://soda-inria.github.io/ken_embeddings/
 
+.. |Pipeline| replace::
+    :class:`~sklearn.pipeline.Pipeline`
+
+.. |OneHotEncoder| replace::
+    :class:`~sklearn.preprocessing.OneHotEncoder`
 
- .. |Pipeline| replace::
-     :class:`~sklearn.pipeline.Pipeline`
+.. |ColumnTransformer| replace::
+    :class:`~sklearn.compose.ColumnTransformer`
 
- .. |OneHotEncoder| replace::
-     :class:`~sklearn.preprocessing.OneHotEncoder`
+.. |MinHashEncoder| replace::
+    :class:`~skrub.MinHashEncoder`
 
- .. |ColumnTransformer| replace::
-     :class:`~sklearn.compose.ColumnTransformer`
+.. |get_ken_embeddings| replace::
+    :class:`~skrub.datasets.get_ken_embeddings`
 
- .. |MinHash| replace::
-     :class:`~skrub.MinHashEncoder`
+.. |HGBR| replace::
+    :class:`~sklearn.ensemble.HistGradientBoostingRegressor`
 
- .. |HGBR| replace::
-     :class:`~sklearn.ensemble.HistGradientBoostingRegressor`
+.. |FeatureAugmenter| replace::
+    :class:`~skrub.FeatureAugmenter`
 """
 
 ###############################################################################
-# The data
-# --------
+# Predicting video game sales
+# ---------------------------
 #
-# We will take a look at the video game sales dataset.
-# Let's retrieve the dataset:
+# In this example, we will take a look at a video game sales dataset.
+# We first retrieve our base dataset:
+
 import pandas as pd
 
 X = pd.read_csv(
     "https://raw.githubusercontent.com/William2064888/vgsales.csv/main/vgsales.csv",
     sep=";",
     on_bad_lines="skip",
 )
-# Shuffle the data
-X = X.sample(frac=1, random_state=11, ignore_index=True)
 X.head(3)
 
 ###############################################################################
-# Our goal will be to predict the sales amount (y, our target column):
+# Our goal will be to predict our target column y, the global sales
+# (in millions of copies, known via
+# `cross-sourcing <https://www.gamecubicle.com/features-mario-units_sold_sales.htm>`_).
+
 y = X["Global_Sales"]
 y
 
 
 ###############################################################################
 # Let's take a look at the distribution of our target variable:
+
 import matplotlib.pyplot as plt
 import seaborn as sns
 
 sns.set_theme(style="ticks")
 
-sns.histplot(y)
+fig, ax = plt.subplots()
+sns.histplot(y, kde=True)
+ax.set(xlim=(0, 10), ylim=(0, 1000))
 plt.show()
 
 ###############################################################################
-# It seems better to take the log of sales rather than the absolute values:
-import numpy as np
-
-y = np.log(y)
-sns.histplot(y)
-plt.show()
-
-###############################################################################
-# Before moving further, let's carry out some basic preprocessing:
+# FIXME
```

Comment:
?
```suggestion
```

Reply:
Yeah, means I haven't finished the example :p